Recommend SMCAMDProcessor for AMD CPU Temps (#326)
dortania/OpenCore-Install-Guide: AMD/zen.md

### Add

> Here's where we specify which kexts to load, in what specific order to load, and what architectures each kext is meant for. Make sure that AMDRyzenCPUPowerManagement comes *before* SMCAMDProcessor if you choose to use them, as SMCAMDProcessor depends on it. Otherwise, we recommend leaving what ProperTree has done by default, however for 32-bit CPUs please see below:

Reviewer: Does Propertree not get the order correct already? If the kext is defined as needed under the Info.plist, ProperTree should be able to detect this by default.

Reviewer 2: ProperTree does get the order correct for SMCAMDProcessor.

::: details More in-depth Info

The main thing you need to keep in mind is:

* Load order
  * Remember that any plugins should load *after* its dependencies

>   * This means kexts like Lilu **must** come before VirtualSMC, AppleALC, WhateverGreen, the kext AMDRyzenCPUPowerManagement before SMCAMDProcessor (if you included them), etc

Reviewer: ```suggestion
  * This means kexts like Lilu **must** come before VirtualSMC, AppleALC, WhateverGreen, etc. Additionally, AMDRyzenCPUPowerManagement should come before SMCAMDProcessor if you include them.
```
